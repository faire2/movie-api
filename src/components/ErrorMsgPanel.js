import React from "react";

export const ErrorMsgPanel = (props) => {
    const outerStyle = {
        color: "white",
        width: "100%",
        height: "100%",
        display: "flex",
        alignItems: "center",
        justifyContent: "center"
    };

    const innerStyle = {
        backgroundColor: "#f16d8d",
        padding: "1vw",
        borderRadius: "2vw",
    };

    return (
        <div style={outerStyle}>
            <div style={innerStyle}>
            {props.message}
            </div>
        </div>
    )
};