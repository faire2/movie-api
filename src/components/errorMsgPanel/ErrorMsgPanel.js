import React from "react";
import {ErrorPanel, InnerWrapper} from "./errorMsgPanelStyles";

export const ErrorMsgPanel = (props) => {
    return (
        <ErrorPanel>
            <InnerWrapper>
                {props.message}
            </InnerWrapper>
        </ErrorPanel>
    )
};