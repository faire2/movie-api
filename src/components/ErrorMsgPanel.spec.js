import React from 'react';
import renderer from 'react-test-renderer';
import {ErrorMsgPanel} from "./ErrorMsgPanel";

// snapshot test
describe("Error panel", () => {
    test("snapshot", () => {
        const component = renderer.create(<ErrorMsgPanel message={"Test message"}/>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot()
    })
});