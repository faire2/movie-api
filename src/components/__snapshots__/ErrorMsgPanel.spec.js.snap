// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error panel snapshot 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "color": "white",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "#f16d8d",
        "borderRadius": "2vw",
        "padding": "1vw",
      }
    }
  >
    Test message
  </div>
</div>
`;
